api/src/util/project.ts

```diff
@@ -34,15 +34,31 @@ function getUserProjectConfigPath() {
   return getConfigPath(projectPath || path.join(getStudioPaths().base, '..'))
 }
 
-export function getUserProjectConfig() {
+export const getUserProjectConfig = () => {
   return getConfig(getUserProjectConfigPath())
 }
 
-export function getUserProjectPaths() {
+export const getUserProjectApiConfig = () => {
+  return getUserProjectConfig().api
+}
+
+export const getUserProjectWebConfig = () => {
+  return getUserProjectConfig().web
+}
+
+export const getUserProjectStudioConfig = () => {
+  return getUserProjectConfig().studio
+}
+
+export const getUserProjectBrowserConfig = () => {
+  return getUserProjectConfig().browser
+}
```

Comment:
I'm not sure I see the value in all of these. But I might be convinced about them if you explain what your reasoning was here 🙂

Reply:
We need `getUserProjectStudioConfig` because we need to show the user's User Impersonation settings.

I added the thinking I might show all of the User Project config like redwood.toml browser.

In future, might be able to read/write to manage toml config via studio.

But can remove if want.